source/connect/connection-options.txt

```diff
@@ -0,0 +1,352 @@
+.. _php-connection-options:
+
+==========================
+Specify Connection Options
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, configure
+
+Overview
+--------
+
+This page describes the MongoDB connection and authentication options
+available in the {+driver-short+}.
+
+Set Connection Options
+----------------------
+
+You can configure your connection by specifying options in the connection URI or by
+passing them to the ``MongoDB\Client`` constructor.
+
+.. _php-connection-uri:
+
+Using the Connection URI
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you pass a connection URI to the ``MongoDB\Client`` constructor, you can include
+connection options in the URI as ``<name>=<value>`` pairs. In the following example,
+the connection URI contains the ``tls`` option with a value of ``true`` and the
+``tlsCertificateKeyFile`` option with a value of ``/path/to/file.pem``:
```

Comment:
S: to avoid "with":
```suggestion
the connection URI sets the ``tls`` option to ``true`` and the
``tlsCertificateKeyFile`` option to ``/path/to/file.pem``:
```